app/client/components/dashboard/metricsBar.jsx

```diff
@@ -81,6 +81,7 @@ const MetricWrap = styled.div`
 `;
 
 const Metric = styled.p`
+  text-align: center;
```

Comment:
minor formatting to get the '132222 SARAFU' to centre align like the 'TOTAL DISTRIBUTED' etc labels. We should probably format it so that 'SARAFU' is always on its own line, but that requires a bit more work so meh